gen3/bin/kube-setup-hatchery.sh

```diff
@@ -82,6 +82,15 @@ if ! g3kubectl get sa "$saName" -o json | jq -e '.metadata.annotations | ."eks.a
     gen3 awsrole attach-policy "arn:aws:iam::aws:policy/AWSResourceAccessManagerFullAccess" --role-name ${roleName} --force-aws-cli || exit 1
 fi
 
+if [[ -f "$(gen3_secrets_folder)/prisma/apikey.json" ]]; then
```

Comment:
curious: why not use `g3auto` to setup this secret?

Reply:
I just followed what we are already doing for Datadog as that is closely resembling the usecase for this. We could use g3auto as well.

Reply:
oh okay

I think it makes more sense to setup this secrets in `kube-setup-secrets` because it can be used by system-wide services so it is not Hatchery specific, but I'll leave the decision to PEs

Reply:
Hmm. I think you're right, I'll update this PR.

Reply:
If we find another usecase for prisma we can move it out of here, so we can get this merged.